lib/transform.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
```

Comment:
I'm happy to split this into a package if you want - what do you reckon would be a good name?